Docs for Linear Pool Subgraph Integrations (#79)
balancer-labs/docs: docs/guides/boosted-pool-creators/new-protocol.md

If all goes well, you should only need to change the fields highlighted above.

## Subgraph

> Adding new integrations to the subgraph is important for tracking linear pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 

Reviewer: ```suggestion
Adding new integrations to the subgraph is important for tracking Linear Pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
```

> In the `abi` folder add two JSON files: one for the new Linear Pool ABI and one for the the corresponding factory ABI. 

Reviewer: ```suggestion
In the `abi` folder, add two JSON files: one for the new Linear Pool ABI and one for the the corresponding factory ABI. 
```

> In the `abi` folder add two JSON files. One containing the new linear pool factory's abi and the next containing the new linear pool's abi. 

Reviewer: ```suggestion
In the `abi` folder, add two JSON files: one for the new Linear Pool ABI and one for the corresponding factory ABI.
```

> Open the `networks.yaml` file and add an entry for each network that the new linear pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`. 

Reviewer: ```suggestion
In `networks.yaml`, add an entry for each network that the new Linear Pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`. 
```

> Add the linear pool to the `PoolType` namespaces within `src/mappings/helpers/pool.ts`. In the `pool.ts` file, update `hasVirtualSupply` and `isLinearPool` to include the new `PoolType`. 

Reviewer: ```suggestion
Add the Linear Pool to the `PoolType` namespaces within `src/mappings/helpers/pool.ts`. In `pool.ts`, update `hasVirtualSupply` and `isLinearPool` to include the new `PoolType`. 
```

> In `manifest.template.yaml`, copy and paste the `ERC4626LinearPoolV4Factory` entry and change to fit the new linear pool's protocol and version. Verify the events match the source code. 

Reviewer: ```suggestion
In `manifest.template.yaml`, copy and paste the `ERC4626LinearPoolV4Factory` entry and change to fit the new Linear Pool's protocol and version. Verify the events match the source code. 
```

> Add event handler mappings for the the factory in `poolFactory.ts`. Copy and paste `handleNewERC4626LinearPoolV4` and make necessary changes for the new linear pool's protocol and version. The function name must match the name provided in the factory's handler from `manifest.template.yaml`.

Reviewer: ```suggestion
Add event handler mappings for the factory in `poolFactory.ts`. Copy and paste `handleNewERC4626LinearPoolV4` and make necessary changes for the new Linear Pool's protocol and version. The function name must match the name provided in the factory's handler from `manifest.template.yaml`.
```
